pkg/ottl/ottlfuncs/README.md

```diff
@@ -424,6 +449,50 @@ Examples:
 
 - `ParseJSON(body)`
 
+### SHA1
+
+`SHA1(value)`
+
+The `SHA1` Converter converts the `value` to a sha1 hash/digest.
+
+The returned type is int64.
+
+The input `value` types:
+* string. If it fails then nil will be returned.
+
+If `value` is another type or parsing failed nil is always returned.
+
+The `value` is either a path expression to a telemetry field to retrieve or a literal.
```

Comment:
```suggestion
The returned type is string.

`value` is either a path expression to a string telemetry field or a literal string. If `value` is another type an error is returned.

If an error occurs during hashing it will be returned.
```